wicket-core/src/main/java/org/apache/wicket/Session.java

```diff
@@ -126,6 +127,11 @@
 	{
 		private static final long serialVersionUID = 1L;
 	};
+	/** records if pages have been unlocked for the current request */
+	public static final MetaDataKey<Boolean> IS_RTL = new MetaDataKey<>()
```

Comment:
My suggestion in the Wicket-Bootstrap issue to use the metadata was good if this functionality was in W-B repo, because there there is no way to add a new field.
If we are going to add this functionality to wicket-core then we could use a normal field.

Reply:
In addition to using a field we should also add a non-final getter method. This way an application can define its own logic for deciding whether to use RTL or not.

Reply:
done